test/cases/testdata/jsonremove/test-jsonremove-0253.yaml

```diff
@@ -1,5 +1,5 @@
 cases:
-- data: {}
+- data:
```

Comment:
👀 learned something new about YAML today.

Reply:
yeah it also doesn't matter 😄